docker/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Run tests and make jar
+# Uses docker.
+
+set -eux
+
+# check if mysql is running
+mysql_running=$(docker inspect -f {{.State.Running}} mysql)
+if ! $mysql_running ; then
```

Comment:
After removing the `mysql` container, `docker inspect` also failed because it was missing.